tools/calendar_generation.py

```diff
@@ -0,0 +1,61 @@
+import calendar
+import datetime
+
+
+class CalendarTime():
+
+    def __init__(self, monthName, currentYear, currentMonth):
+        self.monthname = monthName
+        self.year = currentYear
+        self.month = currentMonth
+
+
+def generate_month(CalTimeObject):
+    """
+    Generate month's calendar as a matrix
+
+    :param CalTimeObject: Object with month, year, name of the month
+    :return: Matrix 6x7 with a days in a month
+    :rtype: list
+    """
+    calendar_ret = calendar.monthcalendar(CalTimeObject.year, CalTimeObject.month)
```

Comment:
This function returns lists of the same length, so when in first week month has only 3 days (like November where 1,2,3 are at the end of the week) then ot will fill the rest of days with 0 `[0, 0, 0, 0, 1, 2, 3]`.
As the result there are buttons with 0 value